lib/plugins/aws/deploy/compile/events/apiGateway/lib/validate.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
           let path;
 
           if (typeof event.http === 'object') {
-            method = event.http.method;
+            method = event.http.method.toLowerCase();
```

Comment:
You are completely correct, I've implemented a fix for this to also test both. Imho its nicer to have the toLowerCase as separate calls to make it clear in the one line whats happening. Doubles the code, but imho is a little easier to understand.